### ISSUE TITLE
Contrast Minimum Accessibility

### ISSUE THREAD

**Issue by shrey-sachdeva2000** [accordproject/techdocs#268]
### **WCAG 2.0 Level AA 1.4.3 (Contrast (Minimum))**

**Tasks**
Contrast ratio should be at least 4.5:1.

**Related Issues:**
Issue #57 opened on Jun 13, 2019 by @irmerk  in cicero-ui

**Description**
The color of body text is low contrast(3.55:1) on grey background(#f9f9f9) failing to match the minimum requirement of at least 4.5:1.
![contrast_issue](https://user-images.githubusercontent.com/47667325/75091299-cad81880-5591-11ea-9a25-b5a911bd97c5.PNG)

**Solution**
Making the text #24292e would significantly improve the contrast ratio(13.93).
